### PR TITLE
Add support for comments on patchSets - new feature

### PR DESCRIPTION
**Pull request by lukaszgryglicki**
Signed-off-by: Łukasz Gryglicki <lgryglicki@cncf.io>

### CODE REVIEW

File: cmd/gerrit/gerrit.go [hunk 1]
```diff
@@ -648,6 +650,26 @@ func (j *DSGerrit) ConvertDates(ctx *shared.Ctx, review map[string]interface{})
...
+				if ok {
+					comments, ok := iComments.([]interface{})
+					if ok {
+						for commentNum, iComment := range comments {
+							comment, ok := iComment.(map[string]interface{})
+							if !ok {
+								continue
+							}
+							comment["level"] = "patchset"
+							comment["patchset_comment_num"] = fmt.Sprintf("%d:%d", patchNum, commentNum)
```
Comment (ajinkyan83) [inline cmd/gerrit/gerrit.go:666]: Suggestion: `patchset_comment_num`  can be `patchset_comment_index` ?

Comment (lukaszgryglicki) [reply cmd/gerrit/gerrit.go:666]: Changing.

Comment (lukaszgryglicki) [reply cmd/gerrit/gerrit.go]: Updated.

**Review by ajinkyan83 — APPROVED** [review]
/lgtm

**Pull request merged**